extensions/integration-tests/package.json

```diff
@@ -10,7 +10,6 @@
   },
   "activationEvents": [
 		"onFileSystem:memfs",
-    "onDebug",
```

Comment:
Not sure why this was added, but it doesn't seem useful at all. It only causes to have errors that prompt the user to install the postgres extension